src/components/Chat/ChatMessageComponent.tsx

```diff
@@ -7,7 +7,8 @@ interface IProps {
 }
 
 const ChatMessageComponent = (props: IProps) => {
-  const authorType = props.message.author === 'student' ? 'self' : 'other';
+  const { message, author } = props.message;
+  const authorType = author === 'frivillig' ? 'self' : 'other';
```

Comment:
Inneholder author i ITextMessage typen til brukeren og ikke navnet? Burde kanskje hete noe annet og være en enum type i så fall

Reply:
Det er egentlig en placeholder for en brukerID/ navnet til brukeren, men for øyeblikket har vi ingen auth